src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -157,12 +131,23 @@ export const GridRow = forwardRef<HTMLDivElement, GridRowProps>(
               gridLayoutStateManager={gridLayoutStateManager}
               renderPanelContents={renderPanelContents}
               interactionStart={(type, e) => {
-                e.stopPropagation();
-
-                // Disable interactions when a panel is expanded
-                const isInteractive = gridLayoutStateManager.expandedPanelId$.value === undefined;
+                // ignore all interactions when panel is expanded or when not in edit mode
+                const isInteractive =
+                  gridLayoutStateManager.expandedPanelId$.value === undefined &&
+                  gridLayoutStateManager.accessMode$.getValue() === 'EDIT';
                 if (!isInteractive) return;
 
+                // ignore anything but left clicks for mouse events
+                if (isMouseEvent(e) && e.button !== 0) {
+                  return;
+                }
+                // ignore multi-touch events for touch events
+                if (isTouchEvent(e) && e.touches.length > 1) {
+                  return;
+                }
```

Comment:
I realized in my testing that we weren't ignoring right clicks on the resize handle - so, I moved the "ignore" logic that we had in the drag handle to the `interactionStart` callback so that **all** interaction events have the same ignore logic.